image_width: 1280
image_height: 720
camera_name: nexigo_camera
camera_matrix:
  rows: 3
  cols: 3
  data: [644.5166640894647, 0, 633.3565210161793, 0, 643.3742143109026, 325.8863770306952, 0, 0, 1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.090794646725051, -0.06334101239287704, -0.008328537416810737, 0.005849565188551934, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [664.4642944335938, 0, 643.7645792714175, 0, 0, 666.8670654296875, 318.1437043454025, 0, 0, 0, 1, 0]